contrib/mpd/template.py

```diff
@@ -0,0 +1,169 @@
+pkgname = "mpd"
+pkgver = "0.23.15"
+pkgrel = 0
+build_style = "meson"
+# MPD source tree includes a clashing `build` dir
+configure_args = [
+    "-Ddocumentation=enabled",
+    "-Dhtml_manual=true",
+    "-Dmanpages=true",
+    "-Dsyslog=enabled",
+    "-Dinotify=true",
+    "-Dio_uring=enabled",
+    "-Ddaemon=true",
+    "-Dsystemd=disabled",
+    "-Depoll=true",
+    "-Deventfd=true",
+    "-Dsignalfd=true",
+    "-Dtcp=true",
+    "-Dipv6=enabled",
+    "-Dlocal_socket=true",
+    # Audio formats
+    "-Ddsd=true",
+    # Database plugins
+    "-Ddatabase=true",
+    "-Dupnp=disabled",
+    "-Dlibmpdclient=disabled",
+    # Neighbor plugins
+    "-Dneighbor=true",
+    # Storage plugins
+    "-Dudisks=enabled",
+    "-Dwebdav=enabled",
+    # Playlist plugins
+    "-Dcue=true",
+    # Input plugins
+    "-Dcdio_paranoia=enabled",
+    "-Dcurl=enabled",
+    "-Dmms=enabled",
+    "-Dnfs=enabled",
+    # Archive plugins
+    "-Dbzip2=enabled",
+    "-Diso9660=enabled",
+    "-Dzzip=disabled",
+    # Tag plugins
+    "-Did3tag=enabled",
+    "-Dchromaprint=enabled",
+    # Decoder plugins
+    "-Dadplug=disabled",
+    "-Daudiofile=enabled",
+    # FIXME: FFMPEG works but AAC is fairly common and this should be packaged
```

Comment:
i think ffmpeg working is fine? nothing wrong with the ffmpeg decoders after all